### PR TITLE
Add Son of Doge (BEP20)

### PR DESCRIPTION
**Pull request by dnt344444**
Adding info for BEP20 token 'Son of Doge'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by cryptomanz** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 5000 holder addresses and 10000 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863).
Closing PR for now. Please come back to us when all criteria are met.

File: blockchains/binance/assets/0xa908417aa4d77c60a3359df7545b5a589118786f/info.json
Comment (BGKDENT) [inline blockchains/binance/assets/0xa908417aa4d77c60a3359df7545b5a589118786f/info.json:1]: assets/blockchains/smartchain/assets/1234abcd...abcd/logo.png
assets/blockchains/smartchain/assets/1234abcd...abcd/info.json

Comment (BGKDENT) [reply blockchains/binance/assets/0xa908417aa4d77c60a3359df7545b5a589118786f/info.json]: assets/blockchains/smartchain/assets/1234abcd...abcd/logo.png
assets/blockchains/smartchain/assets/1234abcd...abcd/info.json

Comment (Japojonesy) [reply blockchains/binance/assets/0xa908417aa4d77c60a3359df7545b5a589118786f/info.json:1]: approved

**Pull request closed without merging**